src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -363,11 +364,6 @@ private void ToggleSidebarCollapsedState(KeyboardAcceleratorInvokedEventArgs? e)
 			e!.Handled = true;
 		}
 
-		private void SidebarControl_Loaded(object sender, RoutedEventArgs e)
-		{
-			SidebarAdaptiveViewModel.UpdateTabControlMargin(); // Set the correct tab margin on startup
```

Comment:
Is this no longer used?

Reply:
UpdateTabControlMargin only updates a never-read value. Dead code is removed.